packages/styles/src/banner/README.md

```diff
@@ -0,0 +1,45 @@
+# Banner
+
+## Default
+
+```html
+<div class="as-banner">
+  <div class="as-banner__icon">
+    <i class="as-info"></i>
+  </div>
+  <div class="as-banner__content">This is the default banner without any support colors</div>
+</div>
+```
+
+## For notifications
+
+```html
+<div class="as-banner as-bg--success as-color--type-04">
+  <div class="as-banner__icon">
+    <i class="as-tick-circle-fill"></i>
+  </div>
+  <div class="as-banner__content">Your profile settings have been updated successfully</div>
+</div>
+```
+
+## For warnings
+
+```html
+<div class="as-banner as-bg--warning as-color--type-04">
+  <div class="as-banner__icon">
```

Comment:
what happens if we want to use a `img` ?

Reply:
you can use whatever you want, inside that div